packages/api/endpoints/elasticsearch.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const router = require('express-promise-router')();
+
+const log = require('@cumulus/common/log');
+
+const mappings = require('../models/mappings.json');
+const { defaultIndexAlias, Search } = require('../es/search');
+
+// const snapshotRepoName = 'cumulus-es-snapshots';
+
+async function createEsSnapshot(req, res) {
+  return res.boom.badRequest('Functionality not yet implemented');
+
+  // *** Currently blocked on NGAP ****
```

Comment:
So we're leaving this code here to finish implementing once we're not blocked by NGAP?

Reply:
Yeah I want to implement it once the changes get put in